snapcraft/internal/remote_build/_worktree.py

```diff
@@ -145,7 +145,10 @@ def _archive_part_sources(self, part_name: str, selector=None):
         os.makedirs(os.path.split(archive_path)[0], exist_ok=True)
         command = ["tar", "czf", archive_path, "-C", source_path, "."]
         subprocess.check_call(command)
-        return os.path.relpath(archive_path, self._repo_dir)
+        relpath = os.path.relpath(archive_path, self._repo_dir)
```

Comment:
you could make pathlib.Path objects out of this and return .to_posix ... Path will work appropriately depending on platform

Reply:
done (as_posix)